src/_css/ItemsNotFoundComponent.scss

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css?family=Mali');
```

Comment:
My advice is that use the font-family we currently have in the repository for uniformity across the app.  Ask Arthur about the font family being used